XiEditor/EditViewController.swift

```diff
@@ -505,19 +539,41 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
         }
         dragEvent = theEvent
     }
-    
+
     override func mouseUp(with theEvent: NSEvent) {
         dragTimer?.invalidate()
         dragTimer = nil
         dragEvent = nil
     }
-    
+
+    override func mouseMoved(with theEvent: NSEvent) {
```

Comment:
is this supposed to do anything? I can't seem to get it to work.

Reply:
This is the one part that I don't think this implementation would do justice. In order to get us a real hover, some more work that hooks on text is required. For now, Option+Click is the most reliable option when trying to get a hover to work.